data/db-schema/src/main/resources/net/corda/db/schema/rbac/db.changelog-master.xml

```diff
@@ -2,5 +2,5 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
-    <include file="net/corda/db/schema/rbac/migration/rpc-rbac-creation-v1.0.xml"/>
+    <include file="net/corda/db/schema/rbac/migration/rest-rbac-creation-v1.0.xml"/>
```

Comment:
RBAC isn't only for REST so this should be just "rbac-creation-v1.0.xml" IMO

But I wouldn't be against just reverting the rbac schema changes as they probably aren't top priority.